table.go

```diff
@@ -302,6 +304,69 @@ func (t *Table) SetBorders(border Border) {
 	t.borders = border
 }
 
+// SetStructs sets header and rows from slice of struct.
+// If something that is not a slice is passed, error will be returned.
+// The tag specified by "tablewriter" for the struct becomes the header.
+// If not specified or empty, the field name will be used.
+func (t *Table) SetStructs(v interface{}) error {
+	vt := reflect.TypeOf(v)
+	vv := reflect.ValueOf(v)
+	switch vt.Kind() {
+	case reflect.Slice, reflect.Array:
+		if vv.Len() < 1 {
+			return errors.New("empty value")
+		}
+		first := vv.Index(0)
+		e := first.Type()
+		switch e.Kind() {
+		case reflect.Struct:
+			// OK
```

Comment:
what exactly is going to happen here?

Reply:
Jump to L332 and continue processing.
If I misunderstood the intent of your question, please add a comment.